src/pages/_app.js

```diff
@@ -1,10 +1,45 @@
-import GlobalStyle from "../../styles";
+import { useState } from "react";
+import { useRouter } from "next/router";
+import transactionsData from "@/src/data/transactions.json";
 
 export default function App({ Component, pageProps }) {
+  const [transactions, setTransactions] = useState(transactionsData);
+  const [mode, setMode] = useState("default");
+  const [action, setAction] = useState("default");
+  const router = useRouter();
+
+  function handleMode(mode) {
+    setMode(mode);
+  }
+  function handleCancel() {
+    handleMode("default");
+  }
+
+  function handleConfirmDelete() {
+    console.log("handleConfirmDelete");
```

Comment:
I would recommend to delete all `console.log` straight after testing.

Reply:
valid point!